IFTB.bs

```diff
@@ -0,0 +1,535 @@
+uint8         tableCount  - the number of tables the chunk has data for (must be 1 or 2)
+uint16        GIDs[glyphCount]    - an array of GIDs included in the chunk
+uint32        tables[tableCount]  - an array of tables (by tag) included in the chunk.
+Offset32      offsets[glyphCount * tableCount] - an array of offsets of glyph data,
+                                                 with the first glyphCount offsets corresponding
+                                                 to table 1 and (if present) the second glyphCount
+                                                 offsets corresponding to table 2. All offsets are
+                                                 from the start of the table.
+uint8         glyphData[varies]  - the actual glyph data picked out by the offsets.
+```
+
+<code>tableCount</code> will be 2 only when there is both a <code>glyf</code> and a <code>gvar</code>
+table.  Otherwise <code>tableCount</code> will be 1 and the table will be <code>CFF </code>
+<code>CFF2</code> or <code>glyf</code>
+
+### The Compressed Chunk Format (IFTZ) ### {#iftz-format}
+
+A compressed chunk string is the same as an uncompressed chunk string except:
+
+1.  The <code>version</code> is <code>IFTZ</code>.
+2.  All data after the <code>length</code> field is [[!RFC7932|brotli compressed]]
+
+The compressed chunk contains no indication of its (compressed) length, but that length is encoded
+in the <code>chunkOffsetList</code> of the IFTB table.
+
+Browser Behaviors {#iftb-browser-behaviors}
+-------------------------------------------
+
+### First Request ### {#iftb-browser-behaviors-first-request}
+
+1.   The browser loads a font file and identifies it as an IFTB font by its "sfnt" header version (in
+     the uncompressed case) or by its WOFF2 "flavor" (in the compressed case).  These are the first
+     and second 4-byte words in the files respectively.
+2.   In the compressed case, it decodes the WOFF2 into its corresponding OTF (The browser can 
+     optionally isolate the IFTB and cmap tables early in the stream to proceed before the remaining
+     font content downloads.)
+3.   The browser determines the initial font subset description corresponding to the content it needs
+     to render using the font.  (IFTB does not provide subsetting of variable axes, so this is just
+     the sets of codepoints and layout features.)
+4.   The browser maps each codepoint in the font subset description to a GID using the cmap table. It
+     then maps that GID to a chunk index using GIDMap of the IFTB table. That index is added to a
+     "chunk set" (which could be a bitmap or std::vector<bool> indexed by chunk index.
+5.   The browser then look up each layout feature in the font subset description in the IFTB table
+     featureTable. That table maps the initial GID-mapped chunks to higher-indexed feature-specific
+     chunks. If any chunk in the set maps to a feature-specific-chunk the latter is added to the set.
```

Comment:
Can you clarify here if chunks that are added by a feature are allowed to trigger further additions on later features? This process will need to be very precisely defined so that multiple implementations will always arrive at the same result.